src/api/main.py

```diff
@@ -67,10 +69,12 @@ def run_rwd(lat, lon):
                 shutil.rmtree(output_path)
                 return jsonify(**output)
         except:
+            log.exception('Could not GeoJSON from output')
             return error_response('Could not get GeoJSON from output.')
     except Exception as exc:
+        log.exception('Error running Point_Watershed_Function')
```

Comment:
It's included in the output (see: https://docs.python.org/2/library/logging.html#logging.Logger.exception).